modules/php/States/EndOfLifeTrait.php

```diff
@@ -29,12 +29,14 @@ public function actReactBeer($ids)
   {
     // Play the beer cards picked by player
     $player = Players::getActive();
-    if ($ids != null) {
+    if ($ids === null) {
```

Comment:
I think `if (empty($ids))` could be more appropriate here, therefore more resilient.